versionwarning.js

```diff
@@ -2,15 +2,26 @@
     // adapted 2020-05 from https://scikit-learn.org/versionwarning.js
     const latestStable = '0.20';
     const goodPaths = ['stable', 'dev', latestStable];
+    const devbar_style = [
+        'text-align: center',
+        'padding: 5px',
+        'margin-bottom: 5px',
+        'border-radius: 0 0 4px 4px !important',
+        'background-color: #e74c3c',
+        'border-color: #e74c3c',
+        'color: #ffffff',
+    ].join('; ')
     const showWarning = (msg) => {
-        $('body').prepend(`<div class="d-block devbar alert alert-danger">${msg}</div>`)
+        $('body').prepend(`<div class="d-block devbar alert alert-danger" style="${devbar_style}">${msg}</div>`)
+
     };
     if (location.hostname === 'mne.tools' || location.hostname.includes('circle-artifacts.com')) {
```

Comment:
I think we don't actually need `hostname.includes('circle...')` here anymore, since Circle is almost entirely used for building the *dev* docs, and @larsoner requested that for the dev docs we stick with the devbar definition within `layout.html` rather than via javascript. The way this is currently, [we're getting 2 bars](https://20207-1301584-gh.circle-artifacts.com/0/dev/index.html).  Is there a reason to keep `hostname.includes('circle...')` here that I'm not thinking of?

Reply:
Agreed we probably don't need it here